lib/elixir/lib/kernel/typespec.ex

```diff
@@ -578,7 +578,7 @@ defmodule Kernel.Typespec do
   defp typespec_to_ast({:type, line, :map, fields}) do
     fields = Enum.map fields, fn
       {:type, _, :map_field_assoc, :any} ->
-        {:..., [line: line], nil}
+        {{:optional, [], [{:any, [], []}]}, {:any, [], []}}
```

Comment:
@ericmj I'm not quiet sure if this is the AST for this. Please let me know if it should be something else.